### PR TITLE
Add instruction to add global.json to preview2

### PR DESCRIPTION
**Pull request by jchannon**
skipci

Update the README description about preview4 to show that you can have preview4 installed but you need to set a target in preview2 projects inside a global.json file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The current official release of the csproj-enabled CLI tools is **CLI Preview 4*
 There are a couple of things to keep in mind:
 
 * Preview 4 CLI bits are still **in development** so some rough edges are to be expected. 
-* Preview 4 bits **do not support** project.json so you will have to either keep Preview 2 tools around or migrate your project. You can find more information on this using the [project.json to csproj instructions](https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/ProjectJsonToCSProj.md). 
+* Preview 4 bits **do not support** project.json so you will have to either keep Preview 2 tools around or migrate your project or add a global.json file to your project to target preview2. You can find more information on this using the [project.json to csproj instructions](https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/ProjectJsonToCSProj.md).
```
Comment (svick) [inline README.md]: Shouldn't adding global.json be part of the "kepp Preview 2 tools around" option? I.e.:

> Preview 4 bits **do not support** project.json so you will have to either keep Preview 2 tools around (and add a global.json file to your project to target preview2) or migrate your project.

Comment (jchannon) [inline README.md]: I can change it to that if you prefer

Comment (dasMulli) [inline README.md]: Is there an example `global.json` somewhere that could be linked? might be super helpful. especially if that includes the full version number.

Comment (jchannon) [inline README.md]: It's in the link on the next line

**Comment by blackdwarf** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (dasMulli) [inline README.md]: Of course. silly me 🙃

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Windows_NT x86 Debug Build please.

**Comment by livarcocc** [comment]
@jchannon Thanks for the PR.

**Comment by piotrpMSFT** [comment]
@MattGertz FYI, merged documentation-only PR.

**Pull request merged**